helm/cluster-azure/values.schema.json

```diff
@@ -103,7 +103,76 @@
+                                        "overwrite": false
+                                    },
+                                    {
+                                        "device": "/dev/disk/azure/scsi1/lun1",
+                                        "extraOpts": [
+                                            "-E",
+                                            "lazy_itable_init=1,lazy_journal_init=1"
+                                        ],
+                                        "filesystem": "ext4",
+                                        "label": "containerd_disk",
+                                        "overwrite": false
+                                    },
+                                    {
+                                        "device": "/dev/disk/azure/scsi1/lun2",
+                                        "extraOpts": [
+                                            "-E",
+                                            "lazy_itable_init=1,lazy_journal_init=1"
+                                        ],
+                                        "filesystem": "ext4",
+                                        "label": "kubelet_disk",
+                                        "overwrite": false
+                                    }
+                                ]
+                            },
+                            "ignition": {
+                                "containerLinuxConfig": {
+                                    "additionalConfig": {
+                                        "storage": {
+                                            "disks": [
+                                                {
+                                                    "device": "/dev/disk/azure/scsi1/lun0",
+                                                    "partitions": [
+                                                        {
+                                                            "number": 1
+                                                        }
+                                                    ]
+                                                }
+                                            ]
+                                        }
+                                    }
+                                }
+                            },
+                            "mounts": [
+                                [
+                                    "etcd_disk",
+                                    "/var/lib/etcddisk"
+                                ],
+                                [
+                                    "containerd_disk",
+                                    "/var/lib/containerd"
+                                ],
+                                [
+                                    "kubelet_disk",
+                                    "/var/lib/kubelet"
+                                ]
+                            ],
+                            "preKubeadmCommands": [
```

Comment:
We need that to wait until everything is ready, otherwise kubeadm.sh script might fail which has bigger implications that nodes node coming up properly